bin/codemods/run.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
+	const excludeUtilityFiles = name => name !== 'config.js' && name !== 'helpers.js';
+	const excludeDotFiles = name => name[0] !== '.';
+	const includeJSFiles = name => path.extname( name ) === '.js';
+	return fs.readdirSync( './bin/codemods/src' ).filter(
+		excludeUtilityFiles
+	).filter(
+		excludeDotFiles
+	).filter(
+		includeJSFiles
+	).map( name => name.replace( '.js', '' ) );
```

Comment:
I think these could end up a bit cleaner either defined outside of this function or inlined…

```js
return fs.readdirSync( './bin/codemods/src' )
	.filter( name => name !== 'config.js' && name != 'helpers.js' ) // exclude utility files
	.filter( name => name[ 0 ] !== '.' ) // exclude dot files
	.filter( name => path.extname( name ) === '.js' ) // only include JS files
	.map( name => name.replace( '.js', '' ) ); // strip extension from filename
```

Reply:
👍 I don't mind either way; let me push up a commit.

Reply:
I'd consider using the [glob](https://www.npmjs.com/package/glob) and [ignore](https://www.npmjs.com/package/ignore) libraries. `glob` can list files in a given directory with given extensions. Dot files are ignored by default. The `ignore` library implements everything you need to work with `.gitignore`-like files. In your case, ignore the `config.js` and `helpers.js`.

```js
const ig = ignore().add( [ 'config.js', 'helpers.js' ] );
const files = glob.sync( 'bin/codemods/src/*.js' )
  .filter( name => ig.ignores( name ) )
  .map( name => name.replace( /\.js$/, '' );
```

You can see them in action in the [script that reformats](https://github.com/Automattic/wp-calypso/blob/master/bin/reformat-files.js#L36-L50) files with Prettier. Both libraries are already installed in your Calypso `node_modules`.

Maybe it's an overkill in this case, maybe not.

Reply:
I personally think the `ignore` library is overkill here. However, I feel `glob`'s expressive parameter will enhance readability. Let me push up a commit.